flutter_local_notifications/macos/Classes/FlutterLocalNotificationsPlugin.swift

```diff
@@ -241,6 +243,25 @@ public class FlutterLocalNotificationsPlugin: NSObject, FlutterPlugin, UNUserNot
         }
     }
 
+    func getActiveNotifications(_ result: @escaping FlutterResult) {
+        if #available(OSX 10.14, *) {
+            UNUserNotificationCenter.current().getDeliveredNotificationRequests { (requests) in
```

Comment:
Can you fix the compilation error? The name of the actual method is different. See https://developer.apple.com/documentation/usernotifications/unusernotificationcenter/1649520-getdeliverednotifications. The code to extract the details would also need to change as a result though should be similar to what you had in the iOS implementation but translated to Swift